src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomComponentSerializationService.cs

```diff
@@ -226,19 +226,23 @@ public override void DeserializeTo(SerializationStore store, IContainer containe
         }
 
         /// <summary>
-        /// The SerializationStore class is an implementation-specific class that stores  serialization data for the component serialization service.  The  service adds state to this serialization store.  Once the store is  closed it can be saved to a stream.  A serialization store can be  deserialized at a later date by the same type of serialization service.   SerializationStore implements the IDisposable interface such that Dispose  simply calls the Close method.  Dispose is implemented as a private interface to avoid confusion.  The IDisposable pattern is provided  for languages that support a "using" syntax like C# and VB .NET.
+        /// The SerializationStore class is an implementation-specific class that stores serialization data for the component serialization service.  The service adds state to this serialization store.  Once the store is closed it can be saved to a stream.  A serialization store can be deserialized at a later date by the same type of serialization service. SerializationStore implements the IDisposable interface such that Dispose  simply calls the Close method.  Dispose is implemented as a private interface to avoid confusion.  The IDisposable pattern is provided for languages that support a "using" syntax like C# and VB .NET.
```

Comment:
Would be good to wrap this long line
```suggestion
        /// The SerializationStore class is an implementation-specific class that stores serialization data for the component serialization service.  
        /// The service adds state to this serialization store.  Once the store is closed it can be saved to a stream.  A serialization store can
        /// be deserialized at a later date by the same type of serialization service. SerializationStore implements the IDisposable interface such
        /// that Dispose  simply calls the Close method.  Dispose is implemented as a private interface to avoid confusion.
        /// The <see cref="IDisposable" /> pattern is provided for languages that support a "using" syntax like C# and VB .NET.
```

Reply:
thanks